src/style.css

```diff
@@ -1 +1,18 @@
 /* styles go here */
+/* * {
+  box-sizing: border-box;
+}
+ */
```

Comment:
There is an unnecessary empty line. You can remove it to keep the code clean and concise.